### DISCUSSION TITLE
How do we update the scraper?

### DISCUSSION THREAD

**Discussion by paulescobar** [UltimaHoarder/UltimaScraper#743] [Q&A] [answered]
New to the Python installation method on Windows. I installed the scraper a month back (using the "pip install -r requirements.txt" command). I haven't run the program in a while. But today I saw an update is available.

How do I update the scraper without losing my previous edits to the JSON files? Is there some update command? (I couldn't find any in the "Readme").

**Comment by UltimaHoarder** [comment, marked as answer]
Download the latest release or commit and overwrite new files over the old
